### DISCUSSION TITLE
hold for menu button may not respond to operation

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#31135] [Q&A]
I'm playing with a touchscreen on an Arm64 Linux, and I compiled the game myself. I'm not sure if it's a quirk of the ARM device or if there was something wrong with my compilation.
Game Ver: https://github.com/ppy/osu/commit/c93b6dba2b450265fcb832e8c93de49f8f64b495

https://github.com/user-attachments/assets/0111360b-39e2-486e-b7b1-b22ccd567f54

**Comment by peppy** [comment]
I'd say something at a system level is blocking long holds and you're going to have to figure out what that is. Or use windows, it should work fine there.
